frontend/src/components/SeedlotRegistrationSteps/ExtractionAndStorageStep/styles.scss

```diff
@@ -0,0 +1,56 @@
+@use '@carbon/type';
+// Light Theme/Notifications/$notification-info-background
+@use '../../../theme/colors.scss' as colors;
+@use '../../../theme/variables.scss' as vars;
+
+.seedlot-registration-forms.bcgov--row {
```

Comment:
Use `#{vars.$bcgov-prefix}` instead of `bcgov` directly, please fix on other places in this file.